linters/README.md

```diff
@@ -5,3 +5,5 @@ Files to configure the linters we use for each language and technology
 
 * Ruby
   * [Rubocop](/linters/ruby/.rubocop.yml)
+* JavaScript
```

Comment:
The same question applies to rubocop: what if you get a client that is using an old version of ruby? The syntax changed a little bit since the first versions, I guess this rubocop configuration wouldn't work.

I'm aware there are scenarios where this does not fit, but it does not justify including them in the guides. Those are the exceptions, for them we adapt the configuration, but we shouldn't make them a rule in the guides. This is the configuration I think we should always try to use.